pkg/api/fs.go

```diff
@@ -948,7 +966,7 @@ func (f *Fs) GetCache(req *pb.GetCacheRequest, stream pb.Fs_GetCacheServer) erro
 	ctx := stream.Context()
 	trace.SpanFromContext(ctx)
 
-	_, err := requireProjectAuth(ctx)
+	err := requireSharedReaderAuth(ctx)
```

Comment:
I didn't want `cached` to have an admin token or any specific project's token as its the only DL component that actually runs on k8s nodes next to sandboxed workloads. So, I added a new token type that can read the cache and that's it.